TestForSignUp/Registrator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    internal static class Registrator
+    {
+        internal static bool RegistrateWithCorrectData(InputGenerator input)
+        {
+            try
+            {
+                var browser = new ChromeDriver();
+                browser.Navigate().GoToUrl("https://localhost:44335/signup");
+                Thread.Sleep(5000);
+                var filler = new ElementFiller(browser);
+                filler.Fill(input.commonString, input.emailString, input.dateString);
+                var buttons = browser.FindElementsByCssSelector("[type=\"button\"]");
+                buttons[1].Click();
+                Thread.Sleep(5000);
+                var submitButton = browser.FindElementByCssSelector("[type=\"Submit\"]");
+                submitButton.Click();
+                Thread.Sleep(5000);
+                var errors = browser.FindElementsByCssSelector("[style=\" color: red;\"]");
+                if (errors.Count == 1) return false;
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+            return true;
+        }
+
+        internal static bool RegistrateWithIncorrectData()
+        {
+            var browser = new ChromeDriver();
+            browser.Navigate().GoToUrl("https://localhost:44335/signup");
+            Thread.Sleep(5000);
+            var filler = new ElementFiller(browser);
+            filler.Fill("A", "A", "1");
+            var buttons = browser.FindElementsByCssSelector("[type=\"button\"]");
+            buttons[1].Click();
+            Thread.Sleep(5000);
+            var errors = browser.FindElementsByCssSelector("[class=\"validation-message\"]");
+            if (errors.Count == 3) return true;
```

Comment:
wow

Reply:
Actually, I don't really understand what is going on here. Some different strings and numbers. Why do we use them? Why in this order

Reply:
first error: incorrect name
second error: incorrect lastname
thrid name: incorrect email

Reply:
buttons[1] because this button has not some distinctive parameters

Reply:
add variables for buttons[1]

Reply:
> first error: incorrect name
> second error: incorrect lastname
> thrid name: incorrect email

If you have to comment your code then something wasn't written right

Reply:
> first error: incorrect name
> second error: incorrect lastname
> thrid name: incorrect email

I meant 
```
if (errors.Count == 4) return true;
return false; 
```
is equal to` return errors.Count == 4`

Reply:
fixed